EIPS/eip-5008.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 5008
+title: ERC-721 Nonce and Metadata Update Extension
+description: Add a nonce property to ERC-721 tokens.
+author: Anders (@0xanders), Lance (@LanceSnow), Shrug <shrug@emojidao.org>
+discussions-to: https://ethereum-magicians.org/t/eip5008-erc-721-nonce-and-metadata-update-extension/8925
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-04-10
+requires: 165, 721
+---
+
+## Abstract
+
+This standard is an extension of [ERC-721](./eip-721.md). It proposes adding a `nonce` property and `MetadataUpdate` event to ERC-721 tokens.
+
+## Motivation
+
+Some orders of NFT marketplace has been attacked and the NFTs have been sold in a lower price than market floor price. One reason is that users transfer NFT to another wallet and then, after a certain period of time, transfer it back to the original wallet, and the order becomes valid again.
+
+This EIP proposes adding an `nonce` property to ERC-721 tokens, and the `nonce` will be changed when transfer. If `nonce` is added to an order, the order can be checked to avoid attacks.
```

Comment:
Will suggest since this PR has been proposed for merge the following is what I noticed.
`nonce is a word` so his EIP proposes adding `an` before `nonce` according to [https://www.merriam-webster.com/dictionary/nonce](nonce).
Thanks for the new improvement.